packages/forklift-console-plugin/src/modules/Providers/views/details/components/CredentialsSection/components/edit/OvirtCredentialsEdit.tsx

```diff
@@ -165,6 +236,21 @@ export const OvirtCredentialsEdit: React.FC<EditComponentProps> = ({ secret, onC
           isDisabled={insecureSkipVerify}
         />
       </FormGroup>
+      {insecureSkipVerify ? (
+        ''
+      ) : (
+        <Hint>
+          <HintBody>
+            <Trans t={t} ns="plugin__forklift-console-plugin">
```

Comment:
trans should include the text, this should look:
`<HintBody>{hintBody}</HintBody>`